swarm/network/stream/common_test.go

```diff
@@ -141,13 +141,13 @@ func netStoreAndDeliveryWithAddr(ctx *adapters.ServiceContext, bucket *sync.Map,
 
 	cleanup := func() {
 		netStore.Close()
-		os.RemoveAll(datadir)
+		localStoreCleanup()
```

Comment:
why not `localStore.Close()` too?

Reply:
netStore.Close() is calling localstore.Close()